src/Sylius/Behat/Context/Ui/Admin/ManagingCountriesContext.php

```diff
@@ -176,7 +176,13 @@ public function thisCountryShouldBeDisabled(CountryInterface $country)
      */
     public function iShouldNotBeAbleToChoose($name)
     {
-        expect($this->createPage)->toThrow(ElementNotFoundException::class)->during('chooseName', [$name]);
+        try {
+            $this->createPage->chooseName($name);
+        } catch (ElementNotFoundException $exception) {
+            return;
+        }
+
+        throw new \DomainException('Choose name should throw an exception!');
```

Comment:
Expect is great for "throws", hope we get it back soon.

How about putting "throw new" in try block?